certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1417 @@
+    "contact":{
+      "name" : "FactSet Workflow Automation Support",
+      "url": "https://www.factset.com/",
+      "email":"WorkflowAutomation@factset.com"
+    },
+    "version": "1.0"
+  },
+  "x-ms-connector-metadata":[
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.factset.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.factset.com/legal"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Finance;Data"
+    }
+  ],
+  "host": "api.factset.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/book-builder-api/book-list": {},
+    "/book-builder-api/template-list": {},
+    "/book-builder-api/get-book-id/{ticker}": {},
+    "/signals/v2/events/headlines": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "requestId": {
+                        "type": "string",
+                        "description": "requestId"
+                      },
+                      "signalId": {
+                        "type": "string",
+                        "description": "signalId"
+                      },
+                      "signalName": {
+                        "type": "string",
+                        "description": "signalName"
+                      },
+                      "theme": {
+                        "type": "string",
+                        "description": "theme"
+                      },
+                      "source": {
+                        "type": "string",
+                        "description": "source"
+                      },
+                      "eventId": {
+                        "type": "string",
+                        "description": "eventId"
+                      },
+                      "eventDate": {
+                        "type": "string",
+                        "description": "eventDate"
+                      },
+                      "summary": {
+                        "type": "string",
+                        "description": "summary"
+                      },
+                      "userRelevanceScore": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "userRelevanceScore",
+                        "title": ""

```

Comment:
@krithsai Empty property values can be removed until unless you have a specific reason to keep them.